bindings/python/crocoddyl/utils/map-converter.hpp

```diff
@@ -27,7 +27,7 @@ namespace bp = boost::python;
  * \sa Pickle
  */
 template <typename Container>
-struct PickleMap : public PickleVector<Container> {
+struct PickleMap : public eigenpy::PickleVector<Container> {
```

Comment:
I am not familiar with the latest `Eigenpy` development. However, it might be worth moving the `PickleMap` to it. Do you agree or not with it? If not, I would appreciate the reasoning.